packages/cli/cmd/application/main.go

```diff
@@ -77,15 +101,20 @@ func BuildCommandDocsForHugo(cmd *cobra.Command, dir string) error {
 
 func buildRootCmd() *cobra.Command {
 	vars := mainVars{}
+	formatVars := formatVars{}
 	cmd := &cobra.Command{
 		Use:   "agc",
 		Short: shortDescription,
 		Example: `
   Displays the help menu for the specified sub-command.
   /code $ agc account --help`,
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
-			format.SetFormatter(format.FormatterType(vars.format))
+			opts, err := newFormatOpts(formatVars)
+			if err != nil {
+				log.Error().Err(err)
+			}
 			setLoggingLevel()
```

Comment:
This should be the *first* thing we do in this method, in order to guarantee consistent logging.